src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -744,8 +744,9 @@ export default class OrgPerson extends Mixins(CommonMixin, OrgPersonMixin) {
 
   /** Returns True if person name or org name has changed from its original properties. */
   protected hasNameChanged (orgPerson: OrgPersonIF): boolean {
+    // This check is only for firm corrections and does not apply to corps corrections
     // is this a pre-existing person?
-    if (this.isPreExisting && this.isPerson) {
+    if (!this.isBenBcCccUlcCorrectionFiling && this.isPreExisting && this.isPerson) {
```

Comment:
I think this is the correct condition to check. Tested Conversion, Change, Extension, Conversion filings.

Reply:
Resolving as the logic looks correct now.